docs/developer/channels.mdx

```diff
@@ -0,0 +1,272 @@
+- [Collection](developer/api-reference/objects/collection-channel-listing.mdx) 
+- [Product](developer/api-reference/objects/product-channel-listing.mdx)
+- [ProductVariant](developer/api-reference/objects/product-variant-channel-listing.mdx)
+- [Sale](developer/api-reference/objects/sale-channel-listing.mdx)
+- [ShippingMethod](developer/api-reference/objects/shipping-method-channel-listing.mdx)
+- [Voucher](developer/api-reference/objects/voucher-channel-listing.mdx)
+
+
+## Permissions
+
+Listing channels is allowed only for users with active [`is_staff`](developer/api-reference/objects/user.mdx#isactive-boolean) flag.
+
+Creating, editing, removing channel the `MANAGE_CHANNELS` permission is required.
+To edit for example Product availability in Channel, no additional permission besides `MANAGE_PRODUCTS` is needed.
+
+## Getting channel details
+
+Use [`channel`](developer/api-reference/queries/channel.mdx) query to get channel details.
+
+Available fields:
+- `id`: Channel object ID
+- `name`: Channel name
+- `isActive`: Flag which changes if channel is available for shop customers.
+- `slug`: When using channel depending queries, slug is used for selecting right one (for example when requesting Product details)
+- `currencyCode`: Currency code used by the channel.
+- `hasOrders`: Returns `true` when there are already created orders using that channel. If thats the only Channel using this currency, you won't be able to remove it
+
+Request:
+```graphql
+query {
+  query Channel($id: ID!) {
+    channel(id: "Q2hhbm5lbDo0MzM=") {
+      id
+      isActive
+      name
+      slug
+      currencyCode
+      hasOrders
+    }
+  }
+}
+```
+
+
+Response:
+```json
+{
+  "data": {
+    "channel": {
+      "id": "Q2hhbm5lbDo0MzM=",
+      "isActive": false,
+      "name": "Mobile",
+      "slug": "mobile",
+      "currencyCode": "PLN",
+      "hasOrders": false
+    }
+  }
+```
+
+
+## Creating new channel
+
+Creating new channel is done using [`channelCreate`](developer/api-reference/mutations/channel-create.mdx) mutation.
+
+Request:
+```graphql
+mutation {
+  channelCreate(
+    input: {
+      "currencyCode":"PLN",
+      "name":"Mobile",
+      "slug":"mobile"
+    }
+  ){
+    channel {
+      id
+      isActive
+      name
+      slug
+      currencyCode
+    }
+    errors: channelErrors {
+      code
+      field
+      message
+    }
+  }
+}
+```
+
+Response:
+```json
+{
+  "data": {
+    "channelCreate": {
+      "channel": {
+        "id": "Q2hhbm5lbDo0MzM=",
+        "isActive": false,
+        "name": "Mobile",
+        "slug": "mobile",
+        "currencyCode": "PLN"
+      },
+      "errors": []
+    }
+  }
+}
+```
+
+### Channel list
+
+Because some of the channels may be considered non-public (for example - channel for business partners), non-staff users can not use [`channels`](developer/api-reference/queries/channels.mdx) query. 
+
+Request:
+```graphql
+query {
+  channels {
+    name
```

Comment:
Maybe we should add slug as listed field too?

Reply:
I dont think it adds more to the example - fields are described above and in reference